app/helpers/application_helper.rb

```diff
@@ -8,4 +8,8 @@ def page_index params_page, index, per_page
     params_page ||= 1
     (params_page.to_i - 1) * per_page + index + 1
   end
+
+  def correct_restaurant? restaurant
+    return true if @restaurant &.id == restaurant.id
```

Comment:
```suggestion
    return true if @restaurant&.id == restaurant.id
```
sao k dùng cancancan đi em